.github/pull_request_template.md

```diff
@@ -1,3 +1,16 @@
-Please check one of the following, relating to the [CHANGELOG](https://github.com/holochain/holochain-rust/blob/develop/CHANGELOG.md), which should be updated if relevant
-- [ ] my changes to the code affect some exposed aspect of the developer experience, and I have added an item to relevant 'Added, Fixed, Changed, Removed, Deprecated, or Security' heading under the 'Unreleased' heading of the CHANGELOG, with the format `- summary of change [PR#1234](https://github.com/holochain/holochain-rust/pull/1234)`
-- [ ] my changes to the code do not affect any exposed aspect of the developer experience
+## PR summary
+
+## testing/benchmarking notes
+
+( if any manual testing or benchmarking was/should be done, add notes and/or screenshots here )
+
+## followups
+
+( any new tickets/concerns that were discovered or created during this work but aren't in scope for review here )
+
```

Comment:
Here's a suggestion for the docs section.  @Connoropolous any other suggestions?
```suggestion
## documentation

- [ ] this change has no user-facing documentation that needs updating
- [ ] rust-doc comments have been added
- [ ] md-book documentation updates have been added
```

Reply:
This works!

Reply:
Note to self and others, we had a discussion with dhtnetwork discussing how devs could also have the option of delivering documentation for new code in the form of video, if writing is a challenge/pain/undesirable